app/controllers/invoices_controller.rb

```diff
@@ -124,4 +125,9 @@ def invoice_params
         line_item_ids: [],
         line_items_attributes: [:item_id, :line_total, :price, :quantity, '_destroy', :id, :description])
   end
+
+  def is_client_exists?
```

Comment:
rename to `require_client!`. `!` is used because you are changing the state of request ie. redirect